app/flows/check_travel_during_coronavirus_flow/outcomes/_country.erb

```diff
@@ -30,6 +30,8 @@
     </ul>
   <% else %>
     <ul class="govuk-list govuk-list--bullet">
+      <li><a href="https://www.gov.uk/foreign-travel-advice/<%= country.slug %>/entry-requirements#all-travellers" class="govuk-link" rel="noreferrer noopener" target="_blank">all travellers (opens in new tab)</a></li>
```

Comment:
Small query on the link text here - `all travellers` when read out of context doesn't make a lot of sense. Perhaps `advice for all travellers` instead?

(same for the link below, maybe it could be `advice for people who aren't fully vaccinated`?)

Reply:
I've raised this with content.

Reply:
Yeah I thought this too!